<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1217</width>
    <height>898</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MP3 Autotagger</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../media/icon.jpg</normaloff>../media/icon.jpg</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>971</width>
      <height>821</height>
     </rect>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page_autotagger">
     <widget class="QLabel" name="label_mp3_count">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>650</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_find_save_all_tags">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>680</y>
        <width>231</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Find and save tags for all files</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar_find_save_all_tags">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>760</y>
        <width>851</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>0</y>
        <width>851</width>
        <height>641</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout" rowstretch="0,1,0,0,0,0,0,0,0" columnstretch="0,0,0,0" rowminimumheight="0,0,0,0,0,0,0,0,0">
       <item row="3" column="1" colspan="3">
        <widget class="QLineEdit" name="lineEdit_title">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>Title</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>Title</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1" colspan="3">
        <widget class="QLineEdit" name="lineEdit_year">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>Year</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>Year</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="3">
        <widget class="QLabel" name="label_mp3_files_2">
         <property name="text">
          <string>MP3 info</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1" colspan="3">
        <widget class="QLineEdit" name="lineEdit_genre">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>Genre</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>Genre</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="3">
        <widget class="QLineEdit" name="lineEdit_artist">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>Artist</string>
         </property>
         <property name="whatsThis">
          <string/>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="inputMask">
          <string/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>Artist</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="3">
        <widget class="QLineEdit" name="lineEdit_album">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>Album</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>Album</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_mp3_files">
         <property name="text">
          <string>MP3 files list</string>
         </property>
        </widget>
       </item>
       <item row="7" column="3">
        <widget class="QCheckBox" name="checkBox_replace_info">
         <property name="text">
          <string>Replace existing information</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="7" column="2">
        <widget class="QPushButton" name="pushButton_save_tags">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Save tags</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QPushButton" name="pushButton_find_tags">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Find tags</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1" colspan="3">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="lineEdit_mp3_dir">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_browse">
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="1" colspan="3">
        <widget class="QLabel" name="label_cover">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>Click to add new cover</string>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="lineWidth">
          <number>1</number>
         </property>
         <property name="midLineWidth">
          <number>9</number>
         </property>
         <property name="text">
          <string>                                           Cover</string>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0" rowspan="8">
        <widget class="QListWidget" name="listWidget_tracks"/>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_autotagger_status">
      <property name="geometry">
       <rect>
        <x>44</x>
        <y>800</y>
        <width>911</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_youtube2mp3">
     <widget class="QProgressBar" name="progressBar_download">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>710</y>
        <width>871</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_url">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>510</y>
        <width>841</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="dragEnabled">
       <bool>false</bool>
      </property>
      <property name="placeholderText">
       <string>Enter YouTube URL</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_get_audio_info">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>600</y>
        <width>231</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Get audio information</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_download">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>600</y>
        <width>231</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Download</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_options">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>70</y>
        <width>841</width>
        <height>271</height>
       </rect>
      </property>
      <property name="title">
       <string>Audio options</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QComboBox" name="comboBox_audio_select">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>Extension | Bitrate (kbps) | File size (MB)</string>
         </property>
         <property name="statusTip">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_get_bestaudio">
         <property name="text">
          <string>Get best audio</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_convert_audio_to_mp3">
         <property name="text">
          <string>Convert audio to .mp3</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLineEdit" name="lineEdit_video_title">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>400</y>
        <width>641</width>
        <height>31</height>
       </rect>
      </property>
      <property name="toolTip">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="dragEnabled">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
      <property name="clearButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_vide_title">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>370</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Video title</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_video_url">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>480</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Video URL</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_yt_status">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>790</y>
        <width>921</width>
        <height>21</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_tools">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>121</width>
      <height>741</height>
     </rect>
    </property>
    <property name="title">
     <string>Tools</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QPushButton" name="pushButton_app_autotagger">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Autotagger</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_app_youtube2mp3">
       <property name="text">
        <string>Youtube2MP3</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="Line" name="line_vertical">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>0</y>
      <width>20</width>
      <height>831</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1217</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionUpdates"/>
   </widget>
   <widget class="QMenu" name="menuLanguages">
    <property name="title">
     <string>Languages</string>
    </property>
    <addaction name="actionEnglish"/>
    <addaction name="actionSpanish"/>
   </widget>
   <addaction name="menuLanguages"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionSpanish">
   <property name="text">
    <string>Spanish</string>
   </property>
  </action>
  <action name="actionUpdates">
   <property name="text">
    <string>Check for updates</string>
   </property>
   <property name="iconText">
    <string>Check for updates</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
